src/SDKs/DataFactoryV1Extended/Microsoft.Azure.Management.DataFactories.Extended/Microsoft.Azure.Management.DataFactories.Extended.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.DataFactories.Extended</PackageId>
```

Comment:
@nicolaszhang was the previous version called DataFactories.Extended?
Can you explain the reason for the name change?

Reply:
@shahabhijeet does this make sense to you?